wolfcrypt/src/sp_int.c

```diff
@@ -4422,7 +4422,8 @@ static int _sp_cmp(sp_int* a, sp_int* b)
 #endif
 
 #if (!defined(NO_RSA) && !defined(WOLFSSL_RSA_VERIFY_ONLY)) || \
-    !defined(NO_DSA) || defined(HAVE_ECC) || !defined(NO_DH)
+    !defined(NO_DSA) || defined(HAVE_ECC) || !defined(NO_DH) || \
```

Comment:
Please add a line at top of `sp_int.c` documenting `WOLFSSL_SP_521`.

Reply:
Done.